app/index.html

```diff
@@ -105,14 +103,15 @@
     <script src="bower_components/humanize-duration/humanize-duration.js"></script>
     <script src="bower_components/angular-timer/dist/angular-timer.js"></script>
     <script src="bower_components/angular-ui-router/release/angular-ui-router.js"></script>
+    <script src="bower_components/ui-router-extras/release/ct-ui-router-extras.js"></script>
```

Comment:
Just an FYI, you did this right, but I'm not sure it was intentional. If you put imports in the <!-- bowerbuild --> block that are not represented in the dependencies in bower.json, they will get deleted on the next install/build. The reason @JWGmeligMeyling put it below was because of this behaviour, he didn't add it to the bower.json file ;)